app/poros/food.rb

```diff
@@ -0,0 +1,8 @@
+class Food
+  attr_reader :total_hits, :foods_hash
+
+  def initialize(search_results)
+    @foods_list = search_results[:foods]
+    @total_hits = search_results[:totalHits]
+  end
+end
```

Comment:
I don't think you necessarily want to store total hits to a food object, since total hits isn't an attribute of that food object. I think that you could pull out that total hits to get passed back to the controller before passing data to this poro.